src/protocols/aeternity/config.ts

```diff
@@ -36,3 +44,203 @@ export const AE_AENS_DOMAIN = '.chain';
 export const AE_AENS_NAME_MAX_LENGTH = 63 + AE_AENS_DOMAIN.length;
 export const AE_AENS_NAME_AUCTION_MAX_LENGTH = 12 + AE_AENS_DOMAIN.length;
 export const AE_AENS_BID_MIN_RATIO = 1.05;
+
+/**
+ * Estimated time we need to wait for the middleware to sync it's state
+ * with the node. There is a high risk that in some cases this won't be enough
+ * so consider using this constant as a workaround.
+ */
+export const AE_MDW_TO_NODE_APPROX_DELAY_TIME = 5000;
+
+export const AE_HASH_PREFIXES_ALLOWED = [
+  Encoding.AccountAddress,
+  Encoding.Channel,
+  Encoding.ContractAddress,
+  Encoding.Name,
+  Encoding.OracleAddress,
+  Encoding.TxHash,
+] as const;
+
+/**
+ * ITx.function
+ */
+export const TX_FUNCTIONS = {
+  tip: 'tip',
+  retip: 'retip',
+  tipToken: 'tip_token',
+  retipToken: 'retip_token',
+  transfer: 'transfer',
+  deposit: 'deposit',
+  propose: 'propose', // Multisig
+  addLiquidity: 'add_liquidity',
+  addLiquidityAe: 'add_liquidity_ae',
+  removeLiquidity: 'remove_liquidity',
+  removeLiquidityAe: 'remove_liquidity_ae',
+  swapExactTokensForTokens: 'swap_exact_tokens_for_tokens',
+  swapTokensForExactTokens: 'swap_tokens_for_exact_tokens',
+  swapExactAeForTokens: 'swap_exact_ae_for_tokens',
+  swapTokensForExactAe: 'swap_tokens_for_exact_ae',
+  swapExactTokensForAe: 'swap_exact_tokens_for_ae',
+  swapAeForExactTokens: 'swap_ae_for_exact_tokens',
+  createAllowance: 'create_allowance',
+  changeAllowance: 'change_allowance',
+  transferAllowance: 'transfer_allowance',
+  transferPayload: 'transfer_payload',
+  withdraw: 'withdraw',
+  claim: 'claim',
+} as const;
```

Comment:
These are not configs but are constants. Configs are something that changes based on environments.